scripts/release/index.js

```diff
@@ -16,14 +13,33 @@ cooker.run('publish', [
   cook.relatedPackagesByPackage,
   cook.getCurrentVersionByPackage,
   cook.evaluateNewVersionByPackage,
+  cook.byBranch,
```

Comment:
should getLatestReleaseHash depend on branch? how we deal with it for different branches?